src/Cosi-BLS/cosi-handlers.lisp

```diff
@@ -1078,7 +1084,7 @@ bother factoring it with NODE-COSI-SIGNING."
 
 ;; ------------------------------------------------------------------------------------------------
 
-(defun leader-exec (node)
+(defun leader-exec (node prepare-timeout commit-timeout)
```

Comment:
@guitarvydas I would rather see additional arguments beyond the first, primary one be keyword arguments with the defaults being the package special, i.e. 
```(defun leader-exec (node &key (prepare-timeout *cosi-prepare-timeout*) (commit-timeout *commit-timeout*))```
This provides more clues in the source how these variables are being used, allows `leader-exec` to be called without having to remember anything about the order of the parameters beyond the first one.